tests/system/libraries/_chromeArgs.py

```diff
@@ -0,0 +1,28 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2020-2022 NV Access Limited

```

Comment:
A detail: Remove year 2020 for a file that has just been created.

```suggestion
# Copyright (C) 2022 NV Access Limited
```

Reply:
I'm not sure about this. It's the code that is covered by copyright, and the code existed through that whole time.